QUARKUS-1414.md

```diff
@@ -0,0 +1,35 @@
+# QUARKUS-1414 Service Binding improvements
+
+Jira: https://issues.redhat.com/browse/QUARKUS-1414
+
+The goal of this feature is to improve the provided technology preview support of OpenShift Service Binding in Quarkus
+applications.
+
+## Scope of the testing
+
+### Impact on test suites and testing automation
+* We need to ensure all the supported means of producing Quarkus deployments in OpenShift work with service binding
+  * Source S2I
+  * Docker build
+  * Binary S2I
+* Orchestration of OSD will have to be automated to ensure easy testing on OSD with RHOSAK service bindings. This, as of
+  now, is done manually (see QUARKUS-1027).
+* New tests have to be implemented in Quarkus QE test suite, with applications utilizing the following service bindings:
```

Comment:
Are these services sorted by priority? If not pls sort them according the priority / ask for prioritisation on RFE

Reply:
No, there's no priorities, and remember, this still is TP

Reply:
+1